packages/@ourworldindata/core-table/src/OwidTable.test.ts

```diff
@@ -646,3 +646,55 @@ gdp	annotation`
         (table.get("gdp").def as OwidColumnDef).annotationsColumnSlug
     ).toEqual("annotation")
 })
+
+describe("dropEntitiesThatHaveNoDataInAllColumns", () => {
```

Comment:
These tests, as they are now, also pass with `dropRowsWithErrorValuesForAllColumns`. It would be great if you can rewrite them so they capture the differences between the two.